model/finance.php

```diff
@@ -307,9 +283,6 @@ protected function soucinitelAktivit() {
    * @todo hardcode obejití pro vypravěče není dobrý :(
    */
   protected function soucinitelAktivitPozde() {
```

Comment:
Nedá se teoreticky odstranit i tato metoda?

Reply:
Vypadá to, že není potřeba. Jen nevím, jestli se někde neschovává její použití.